sirepo/pkcli/job_agent.py

```diff
@@ -55,14 +57,49 @@ def default_command():
     pkdlog('{}', cfg)
     i = tornado.ioloop.IOLoop.current()
     d = _Dispatcher()
-    def s(n, x):
-        return i.add_callback_from_signal(d.terminate)
+    def s(*args):
+        return i.add_callback_from_signal(_terminate, d)
     signal.signal(signal.SIGTERM, s)
     signal.signal(signal.SIGINT, s)
     i.spawn_callback(d.loop)
     i.start()
 
 
+def start_sbatch():
+    def kill_agent(pid_file):
+        if socket.getfqdn() == pid_file.fqdn:
+            os.kill(pid_file.pid, signal.SIGKILL)
+        else:
+            try:
+                subprocess.check_output(
```

Comment:
why check_output if it isn't checked? Use subprocess.run with `check_returncode=True`.